### ISSUE TITLE
Remove "Login" button from menu bar

### ISSUE THREAD

**Issue by philip-n** [inbaz/ers#287]
As the profile functionality is not implemented yet, remove the "Login" button from menu bar.

Tentatively setting this to goLive - move to later milestone if appropriate.

**Comment by smadamit** [comment]
We decided to keep it at it place. The jugglers can learn -> "Oh there will be a login."

**Comment by philip-n** [comment]
Can we then replace the login page with a "This feature will be implemented later" page?

Currently, as a new user
* I click on "Login" (maybe I have to login first?)
* I don't have login combi yet, so I click "Register"
* There is an error page. Did I do something wrong? :-(
(see #230)

Proposal would be to show a "This feature is not implemented yet. Please check again later!" page instead of the login screen. Would look "nicer" than discovering an error message.

As usual: Just a proposal, feel free to close this issue again if you don't like it!

**Comment by smadamit** [comment]
@christianbross What's your opinion?

**Comment by christianbross** [comment]
I agree with Philip. A page with "sorry - constructions - we're still working on the "login" feature - come back again later, we'll try to have it ready in May"
